src/compiler/types.ts

```diff
@@ -931,12 +930,37 @@ export interface Node extends ReadonlyTextRange {
     /** @internal */ readonly transformFlags: TransformFlags; // Flags for transforms
     /** @internal */ id?: NodeId; // Unique id (used to look up NodeLinks)
     readonly parent: Node; // Parent node (initialized by binding)
-    /** @internal */ original?: Node; // The original node if this is an updated node.
-    /** @internal */ emitNode?: EmitNode; // Associated EmitNode (initialized by transforms)
+    /** @internal */ original?: Node | undefined; // The original node if this is an updated node.
+    /** @internal */ emitNode?: EmitNode | undefined; // Associated EmitNode (initialized by transforms)

```

Comment:
Is this intended to be like `original: Node | undefined`? We don't use `exactOptionalPropertyTypes` (yet?), so this change is a noop.